### ISSUE TITLE
Display pictures in dashboard?

### ISSUE THREAD

**Issue by AlbrechtL** [thingsboard/thingsboard#777]
I plan to send pictures thru MQTT to TB. In #325 is documented how to do this.
Is it possible to show these pictures in the dashboard?

**Comment by RondoRonaldo** [comment]
To store and represent pictures in ThingsBoard you need to encode these pictures to Base64 format first and then, depending on a widget display it.

As an example, having a server attribute "pic" that stores an image in the Base64 format you can use the "image map" widget and adding the required settings to it represent the image.
Here is settings to the widget: 
![image](https://user-images.githubusercontent.com/43723966/84489487-796fa880-acaa-11ea-8d9f-e2e5f2a517f1.png)
![image](https://user-images.githubusercontent.com/43723966/84489516-7eccf300-acaa-11ea-89b2-b702a9292bcb.png)

**Comment by lehnert** [comment]
Hi @RondoRonaldo, I tried following your suggestion to display images on a dashboard. I'm encoding an image as base64 and sending it from a device to my thingsboard. I'm currently creating an image map widget and setting the image URL source entity alias to "Pi Camera" and the Image URL source entity attribute to "picture". Pi Camera is the name of the entity alias for that dashboard of the device "Pi Camera 1" which has a client-side attribute "picture". I have confirmed that the base64 encoding works as I can use the REST api to ask for the picture attribute from that device and I can visualise it in a base64 decoder script. Currently the New Image Map widget just shows up as a blank white image and nothing else. Any help would be appreciated.
